src/app/(post-login)/submission/components/ProgramMenu.tsx

```diff
@@ -74,101 +135,51 @@ const parseGQLResp = (data: SideMenuQuery) => {
 	};
 };
 
-const ProgramMenu = ({ searchQuery }: { searchQuery: string }) => {
-	const params = useParams();
+const MenuContent = ({ programName }: { programName: string }) => {
 	const pathname = usePathname();
-	const activeProgramName = typeof params.shortName !== 'string' ? '' : params.shortName;
+	const pathnameLastSegment = pathname.split('/')[-1];
```

Comment:
I don't think this is working as intended, [-1] is the same as `pathname.split().-1` and returns `undefined`

Reply:
You are correct - not working as intended.
Had it my head I could negative value it like `slice`
Only positive numbers work in array lookup like this

Reply:
https://github.com/icgc-argo/rdpc-ui/pull/108/commits/26a42b9b392a7eb1625cbdc13f1c86e9a5ef87b4